src/System.Composition.Hosting/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,22 @@
+<linker>
+  <assembly fullname="System.Composition.Hosting">
+    <type fullname="System.Composition.Hosting.Providers.ExportFactory.ExportFactoryExportDescriptorProvider">
```

Comment:
It would be useful to add comment about who needs this for all items in these .xml files. (Some of the items have it already, but not all.)